File: src/main/java/com/example/debug/engine/TaskReport.java
package com.example.debug.engine;

/* loaded from: classes.dex */
public class TaskReport {
    private String clientId;
    private String elapsedTime;
    private int nbrCompletedThreads;
    private int nbrPanels;
    private int nbrQueuedThreads;
    private int nbrRunningThreads;
    private int percentageDone;
    private String status;
    private String taskId;

    public String getTaskId() {
        return this.taskId;
    }

    public void setTaskId(String str) {
        this.taskId = str;
    }

    public String getClientId() {
        return this.clientId;
    }

    public void setClientId(String str) {
        this.clientId = str;
    }

    public String getStatus() {
        return this.status;
    }

    public void setStatus(String str) {
        this.status = str;
    }

    public int getNbrRunningThreads() {
        return this.nbrRunningThreads;
    }

    public void setNbrRunningThreads(int i) {
        this.nbrRunningThreads = i;
    }

    public int getNbrQueuedThreads() {
        return this.nbrQueuedThreads;
    }

    public void setNbrQueuedThreads(int i) {
        this.nbrQueuedThreads = i;
    }

    public int getNbrCompletedThreads() {
        return this.nbrCompletedThreads;
    }

    public void setNbrCompletedThreads(int i) {
        this.nbrCompletedThreads = i;
    }

    public int getNbrPanels() {
        return this.nbrPanels;
    }

    public void setNbrPanels(int i) {
        this.nbrPanels = i;
    }

    public int getPercentageDone() {
        return this.percentageDone;
    }

    public void setPercentageDone(int i) {
        this.percentageDone = i;
    }

    public String getElapsedTime() {
        return this.elapsedTime;
    }

    public void setElapsedTime(String str) {
        this.elapsedTime = str;
    }
}

File: src/features/engine/task_report.rs
use std::collections::HashMap;
use serde::{Deserialize, Serialize};


#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct TaskReport {
    pub task_id: Option<String>,
    pub client_id: Option<String>,
    pub status: Option<String>,
    pub nbr_running_threads: i32,
    pub nbr_queued_threads: i32,
    pub nbr_completed_threads: i32,
    pub nbr_panels: i32,
    pub percentage_done: i32,
    pub elapsed_time: Option<String>,
}

impl TaskReport {
    pub fn new() -> Self {
        Self {
            task_id: None,
            client_id: None,
            status: None,
            nbr_running_threads: 0,
            nbr_queued_threads: 0,
            nbr_completed_threads: 0,
            nbr_panels: 0,
            percentage_done: 0,
            elapsed_time: None,
        }
    }
}

impl Default for TaskReport {
    fn default() -> Self {
        Self::new()
    }
}